source/src_pw/test/CMakeLists.txt

```diff
@@ -14,3 +14,16 @@ AddTest(
 )
 
 install(FILES data-H DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+
+AddTest(
+  TARGET hsolver_sum_band
+  LIBS ${math_libs} base cell symmetry md neighbor orb io ions lcao parallel mrrr pdiag pw ri driver xc ELPA::ELPA esolver
+  SOURCES sum_band_test.cpp 
+)
+
+install(FILES STRU DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+install(FILES H.pz-vbc.UPF DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+install(FILES KPT DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+install(FILES SPIN1_CHG DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+install(FILES WAVEFUNC1.txt DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+install(FILES BANDS_1.dat DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
What does file name means, e.g. `data-H`, `WAVEFUNC1.txt`, `BANDS_1.dat`? You can add some more meaningful prefix/suffix to them.